content/developer/rom/dossier/respondents.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Respondents
+status: draft
+---
+
+* TOC
+{:toc}
+
+Respondents are people that fill out questionnaires. The default respondent is the patient, but there's also profess, parent, second_parent, teacher and caregiver.
+
+- In the future second_parent will be merged into parent.
+- You won't see a profess respondent, since you can't send out to professionals, since there isn't usually just one.
```

Comment:
Ik weet niet of deze API ooit door Medo-onderzoekers gebruikt zal worden, maar volgens mij kan het met [Jorn's Professional Respondent](https://github.com/roqua/roqua/pull/1081) wel bij sommige organisaties.